.ci/manywheel/build_cuda.sh

```diff
@@ -114,7 +118,8 @@ if [[ $USE_CUSPARSELT == "1" && $CUDA_VERSION == "11.8" ]]; then
         )
 fi
 
-if [[ $CUDA_VERSION == "12.4" || $CUDA_VERSION == "12.6" ]]; then
+# CUDA_VERSION 12.4, 12.6, 12.8
+if [[ $CUDA_VERSION == 12* ]]; then
     export USE_STATIC_CUDNN=0
     # Try parallelizing nvcc as well
     export TORCH_NVCC_FLAGS="-Xfatbin -compress-all --threads 2"
```

Comment:
Oh! @tinglvv The flags on line 7 ad immediately overwritten here unfortunately...

Reply:
```suggestion
    export TORCH_NVCC_FLAGS="-Xfatbin -compress-all --threads 2 --host-linker-script=use-lcs"